### ISSUE TITLE
Mass Mailing >> Send a Message - Default code being added

### ISSUE THREAD

**Issue by foxation** [justingit/dada-mail#539]
The markdown didn't seem to work with my code! So here's the situation in plain English:

When I begin a test session the editor inserts some default code. I delete that default code at the very beginning of a test session with the Ctrl+A Ctrl+V.

More default code gets inserted at the top of the draft once the first test email has been sent.

Which messes up any other test emails going to different test accounts.

It wouldn't be too much trouble to do Ctrl+A Ctrl+V before sending each test email, but it would be more elegant if Dada Mail would recognize when the contents of the draft message are something other than the default code that Dada Mail inserts so that it wouldn't add the additional code that it adds.

I think it may already be doing something like this since, no matter how many test messages are sent, the additional <html><head><title></title></head><body> only appears once at the top of the page.

**Comment by justingit** [comment]
Speaking of formatting, would you be able to fix the formatting of this issue? I'm having a had time just reading it.

**Comment by foxation** [comment]
I tried and couldn't figure out how to edit it!

I'll take another look.

Gerald Fox

**Comment by foxation** [comment]
It's the pencil!

One moment please ...

**Comment by foxation** [comment]
I've reworded the issue without the code I was trying to insert.

**Comment by foxation** [comment]
Problem solved!

My boilerplate had the <html> and </html> lines in it. It's been like that for a long time. I think it had something to do with Bridge.

Anyway, I've rewritten the code that I cut and paste so that it now begins with <html> and ends with </html>.

There was no problem with my tests after that.

It wasn't causing a problem with the mass mailing. Just with the tests.
